index.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const fsCallback = require("./lib/files-callBack.js");
+const fsPromise = require("./lib/file-promise.js"), write = require("./lib/write");
+const file = process.argv.slice(2)[0];
+
+
+
+/** this is just a function to run 
+ * callBack with two parameters */
+
+function myTestcallBack(err, data) {
+  console.log("hi!")
+}
+/**use callBack read write read
+ * read l
+ * @param file
+ * @param callBack
+ * write
+ * change data field value 
+ * @param file
+ * @param data
+ * @param callBack
+ * read ll
+ * @param file
+ * @param callBack
+ */
```

Comment:
Your JSDoc comments should be about the current function being _defined_, not all the functions called within the current function. So, in this example, the JSDocs comments should just tell me what `useCallbacks` does, what parameters `useCallbacks` requires, and what `useCallbacks` returns. 

https://devhints.io/jsdoc